SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -389,7 +413,7 @@ final class SwedbankPaySDKViewModel: NSObject {
     }
     
     private func getServerSwedbankPayProblem(_ problemType: SwedbankPaySDK.ServerProblem.SwedbankPayProblem, response: Dictionary<String, Any>) -> SwedbankPaySDK.Problem {
-        let subProblems: [SwedbankPaySDK.SwedbankPaySubProblem]? = getSubProblems(response["problems"] as? [Dictionary<String, Any>])
+        let subProblems: [SwedbankPaySDK.SwedbankPaySubProblem]? = getSubProblems(response["problems"] as? [[String: Any]])
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 123 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)